src/chiltepin/jedi/qg/install.py

```diff
@@ -0,0 +1,101 @@
+import textwrap
+
+from parsl.app.app import bash_app, join_app
+
+
+@bash_app(executors=["service"])
+def _clone(env, install_path, tag="develop", stdout=None, stderr=None):
+    return env + textwrap.dedent(
+        f"""
+    echo Started at $(date)
+    echo Executing on $(hostname)
+    git lfs install --skip-repo
+    rm -rf {install_path}/jedi-bundle/{tag}/src
+    mkdir -p {install_path}/jedi-bundle/{tag}
+    cd {install_path}/jedi-bundle/{tag}
+    git clone --branch {tag} https://github.com/JCSDA/jedi-bundle.git src
+    echo Completed at $(date)
+    """
+    )
+
+
+@bash_app(executors=["service"])
+def _configure(env, install_path, tag="develop", stdout=None, stderr=None, clone=None):
+    return env + textwrap.dedent(
+        f"""
+    echo Started at $(date)
+    echo Executing on $(hostname)
+    cd {install_path}/jedi-bundle/{tag}
+    rm -rf build
+    mkdir build
+    cd build
+    # Comment out the parts of the bundle we do not need
+    perl -p -i -e 's/(.* PROJECT vader)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT saber)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT ioda)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT crtm)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT ufo)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT gsw)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT fv3)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT femps)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT soca)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT mom6)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.* PROJECT coupling)/#\\1/g' ../src/CMakeLists.txt
+    perl -p -i -e 's/(.*mpas)/#\\1/ig' ../src/CMakeLists.txt
+    ecbuild -DCMAKE_INSTALL_PREFIX=../ ../src
+    echo Completed at $(date)
+    """
+    )
+
+
+@bash_app(executors=["serial"])
+def _make(
+    env,
+    install_path,
+    tag="develop",
+    jobs=8,
+    stdout=None,
+    stderr=None,
+    configure=None,
+    parsl_resource_specification={"num_nodes": 1},
+):
```

Comment:
More of a nitpick. This section is the only one that puts each parameter on it's own line unlike the functions above. I think if you run black/isort again and push it should fix this.

Reply:
That is the way `black` wants it.  I tried putting parameters on their own lines for the other functions, and `black` didn't like it.  The `black` format test will not pass if I change the formatting.

Reply:
Nevermind.  I figured it out.  After adding a `,` at the end of the argument list, `black` then decided to put all the args on separate lines.

Reply:
I tried it.  Black insists that this is the correct formatting.  I also tried putting each parameter on it's own line for the others, and Black corrected it back to the way I had it.  So, the black tests will not pass if I modify the formatting of this file.